[REQUEST][9.0 & Serverless]: Show flyout history for alerts/host/user/events (#1011)
elastic/docs-content: solutions/security/detect-and-alert/view-detection-alert-details.md

From the right panel, you can also:

* Click **Expand details** to open the [left panel](/solutions/security/detect-and-alert/view-detection-alert-details.md#left-panel), which shows more information about sections in the right panel.

> * Click the history icon (![History icon](/solutions/images/security-history-icon.png "title =15x15")) to display a list of places that you visited from the alert's details flyout. The list shows any rules, alerts, events, users, or hosts who's details you've viewed and can contain up to 10 unique entries. Click an entry in the list to quickly access the object's details.

Author: ```suggestion
* Click the history icon (![History icon](/solutions/images/security-history-icon.png "title =15x15")) to display a list of places that you visited from the alert's details flyout, for example, flyouts for other alerts or users. The list can contain up to 10 unique entries. Click any list entry to quickly access the item's details.
```
